spec/features/toc_spec.rb

```diff
@@ -53,6 +53,34 @@
       # Displays nested component without online content
       expect(page).to have_content "Eddy, Mary P., New Testament Miniature Book, undated"
     end
+
+    it "online toggle switch resets to off when visiting a new collection", js: true do
+      visit "/catalog/C0140"
+
+      # Displays top-level component without online content
+      expect(page).to have_content "Bainbridge, William, Letter to Albert Gallatin, 1820 June 24"
+
+      # Click toggle to show online content only
+      find(".toggle > span").click
+
+      # Does not display top-level component without online content
+      expect(page).not_to have_content "Bainbridge, William, Letter to Albert Gallatin, 1820 June 24"
+
+      new_tab = window_opened_by do
+        page.execute_script('window.open("/catalog/MC221", "_blank");')
+      end
+      within_window new_tab do
+        expect(page).to have_content "Series 1: U.S. diplomacy career, 1900-1978"
+      end
+
+      # Opening a new tab containing the previous collection should preserve that collections's online toggle state
+      page.refresh
+
+      # Displays online content
```

Comment:
Maybe a comment that says you're doing this to make sure that the page rendered after refresh.